src/dotnet/OpenAPI.Validator/Validation/XmsParameterLocation.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.Core.Logging;
+using OpenAPI.Validator.Properties;
+using OpenAPI.Validator.Validation.Core;
+using OpenAPI.Validator.Model;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenAPI.Validator.Validation
+{
+    public class XmsParameterLocation : TypedRule<SwaggerParameter>
+    {
+        private static readonly IEnumerable<string> AllowedGlobalParameters = new List<string>()
+        { "subscriptionid", "api-version", "apiversion" };
```

Comment:
what about "subscription**I**d"

Reply:
I think there's a case insensitive check below, should be okay. A long shot, but check if anyone is also adding parameter name as `subscription-id`